OutOfSchool/OutOfSchool.BusinessLogic/Services/Elasticsearch/ElasticPinger.cs

```diff
@@ -26,7 +26,19 @@ public Task StartAsync(CancellationToken cancellationToken)
     {
         logger.LogInformation("Service started pinging");
         timer = new Timer(
-            cb => IsHealthy = elasticClient.Ping().IsValid,
+            async (cb) =>
```

Comment:
what `cb` means here ?
I cannot see where we're using it? can it be changed to `_` ?

Reply:
Fixed. Thank you!